player/command.c

```diff
@@ -4734,7 +4742,7 @@ static const struct property_osd_display {
     {"ab-loop-b", .msg = "A-B loop: ${ab-loop-a} - ${ab-loop-b}"
                             "${?=ab-loop-count==0: (disabled)}"},
     {"audio-device", "Audio device"},
-    {"hwdec", .msg = "Hardware decoding: ${hwdec-current}"},
+    {"hwdec", .msg = "Hardware decoding: ${hwdec-current}", .queue = true, .prop_name = "hwdec-current"},
```

Comment:
I understand why you do this, but I worry it's little bit too specific to single use-case and cumbersome to maintain for more properties need it. Some properties just print its own value, other query data which may be available asynchronous from the playloop.

Maybe instead of tracking property display queue , they hwdec itself should show a osd message with it changes. For example after decoding error or other event that may trigger that.

Reply:
Yeah it's definitely relatively heavy handed. We could go with something like "trying hwdec/disabling hwdec" instead as a message?

Reply:
I just thought of a way simpler way to accomplish this. I think it works.